code/mobile/android/PhoneVR/app/src/main/cpp/alvr_main.cpp

```diff
@@ -282,7 +282,8 @@ extern "C" JNIEXPORT void JNICALL Java_viritualisres_phonevr_ALVRActivity_render
                 CardboardDistortionRenderer_destroy(CTX.distortionRenderer);
                 CTX.distortionRenderer = nullptr;
             }
-            CTX.distortionRenderer = CardboardOpenGlEs2DistortionRenderer_create();
+            const CardboardOpenGlEsDistortionRendererConfig config{kGlTexture2D};
+            CTX.distortionRenderer = CardboardOpenGlEs2DistortionRenderer_create(&config);
```

Comment:
Is this intended to fix something ?

Reply:
New Cardboard SDK version requires it, otherwise build fails